packages/office-ui-fabric-react/src/components/Icon/Icon.base.tsx

```diff
@@ -65,7 +65,7 @@ export class IconBase extends React.Component<IIconProps, IIconState> {
 
     return (
       <RootType data-icon-name={iconName} {...containerProps} {...nativeProps} className={classNames.root}>
-        {isImage ? <ImageType {...imageProps} /> : children}
+        {isImage ? <ImageType {...imageProps} /> : children ? children : iconContentChildren}
```

Comment:
@dzearing @ecraig12345 I think there are two ways to think about this conditional. One is to do it the way this change is happening above and the other one is to do:

```ts
{children ? children : (isImage ? <ImageType {...imageProps} /> :  iconContentChildren)}
```

I wonder what are your opinion is on which of these two conditionals we should use.

Reply:
I think the first conditional is a little better from a readability perspective, because when you are reading through the ternary operator and evaluating it, you know that it will resolve to either the <ImageType> component or to the children that have been passed through to this component. Then it just becomes which children props it would be resolved to.

Reply:
It's not just a question of readability but of whether images can ever have children. If images *can't* have children, the conditional should stay as-is.

Reply:
Why not just leave it and always add iconContentChildren?

Reply:
I think if Icon ever hand children and it conditionally rendered the children content based on nuance, it would be confusing. I don't see an issue just always doing it.

Reply:
```
{ isImage ? <ImageType {...imageProps} /> : (children || iconContentChildren) }